master/buildbot/test/unit/test_contrib_buildbot_cvs_mail.py

```diff
@@ -107,6 +113,8 @@ def getProcessOutputAndValueWithInput(executable, args, input):
     "similar to getProcessOutputAndValue, but also allows injection of input on stdin"
     d = defer.Deferred()
     p = _SubprocessProtocol(input, d)
+
+    (executable, args) = _encodeExecutableAndArgs(executable, args)
```

Comment:
i know it's equivalent, for *my* eyes it's a bit lighter...  let's consider that as [nit]